### PR TITLE
Clean up smoke tests

### PR DESCRIPTION
**Pull request by trask**

### CODE REVIEW

File: gradle/dependencies.gradle [hunk 1]
```diff
@@ -39,6 +39,8 @@ ext {
             mockito                          : "1.10.19",
             hamcrest                         : "1.3",
             systemRules                      : "1.19.0",
-            logcaptor                        : "2.5.0"
+            logcaptor                        : "2.5.0",
+
+            legacySdk                        : "2.6.0"
```
Comment (heyams) [inline gradle/dependencies.gradle:44]: azureLegacySdk? don't want to get confused with our legacy sdk :)

File: agent/exporter/build.gradle [hunk 2]
```diff
@@ -14,7 +14,7 @@ dependencies {
     // this is needed in order to access AiAppId for appId exchange data
     compileOnly group: 'io.opentelemetry.instrumentation', name: 'opentelemetry-instrumentation-api', version: versions.opentelemetryInstrumentationAlpha
 
-    implementation 'org.slf4j:slf4j-api:1.7.26'
```
Comment (heyams) [inline agent/exporter/build.gradle:17]: i updated this in heya/az-functions branch...

File: gradle/dependencies.gradle (hunk 1 above)
Comment (trask) [reply gradle/dependencies.gradle:44]: this is our legacy sdk, maybe I should rename to `legacySdkForSmokeTests`?

Comment (heyams) [reply gradle/dependencies.gradle:44]: i think legacySdk is fine then.. thought it was azure :(
or aiJavaLegacySdk to make it more specific?

**Review by heyams — APPROVED** [review]

**Pull request merged**